Extensions/PlatformBehavior/tests/FlatFloorPlatformer.spec.js

```diff
@@ -583,8 +587,12 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
         expect(object.getBehavior('auto1').isOnFloor()).to.be(true);
       }
       // is blocked by the wall
-      expect(object.getX()).to.be(wall.getX() - objectWidth);
-      expect(object.getY()).to.be(platform.getY() - objectHeight);
+      // The X scaling of the object is 0.2 which is not a rational number in base 2.
+      // The search is done pixel by pixel on X which doesn't allow to find the right position to stop.
+      // This result to the object being stopped at its position before moving.
+      // It then converge a bit thanks to a lower speed.
+      expect(object.getX()).to.be.within(wall.getX() - objectWidth - 0.3, wall.getX() - objectWidth);
+      expect(object.getY()).to.be.within(platform.getY() - objectHeight - epsilon, platform.getY() - objectHeight + epsilon);
```

Comment:
This is a case we already knew. It wasn't happening here before because the calculus was something like `x * 20 / 10` and now it's `0.2 * x` which give a different result because 0.2 is not rational.

I think it's totally fair. It actually makes this test more interesting.